### PR TITLE
Add iot class per #13933

### PR DESCRIPTION
**Pull request by ghost**
## Proposed change
Add iot class per #13933

### CODE REVIEW

File: source/_integrations/simplisafe.markdown [hunk 1]
```diff
@@ -2,6 +2,7 @@
 title: SimpliSafe
 description: Instructions on how to integrate SimpliSafe into Home Assistant.
 ha_release: 0.81
+ha_iot_class: Cloud Poll
```
Comment (davet2001) [inline source/_integrations/simplisafe.markdown]: Cloud Polling

**Review by davet2001 — CHANGES_REQUESTED** [review]
Other integrations use "Cloud Polling" rather than "Cloud Poll"

File: source/_integrations/simplisafe.markdown (hunk 1 above)
Comment (ghost) [reply source/_integrations/simplisafe.markdown]: Doh!

I think I got the update made.  Please confirm.

**Review by frenck — APPROVED** [review]
Thanks, @SilvrrGIT 👍

**Comment by bachya** [comment]
FYI, this isn't totally true – it's technically Cloud Polling for some attributes, but Cloud Push for things like alarm state. Not the biggest deal.

**Comment by frenck** [comment]
@bachya This has been taken from the configuration flow of SimpliSafe, which in that case if the place where it is wrongly specified.

**Pull request merged**